### PR TITLE
Types/source list map

### PR DESCRIPTION
**Pull request by e-cloud**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header.

If changing an existing definition:
- [x] Provide a URL to  documentation or source code which provides context for the suggested changes: <<https://github.com/webpack/source-list-map>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by e-cloud** [comment]
@yuit the reviewer is the same as the author which is me. I think there is something missing that the dt-bot should not ping reviewers when they are the same guy

File: source-list-map/index.d.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class SourceListMap {
 
     toString(): string;
 
-    toStringWithSourceMap(options: { file: any }): {
+    toStringWithSourceMap(options: { file: any; }): {
```
Comment (Kovensky) [inline source-list-map/index.d.ts]: Typo? This doesn't look like it should be valid

Comment (e-cloud) [inline source-list-map/index.d.ts]: in fact, it's valid. Just no consistant. Resolve soon.

**Comment by e-cloud** [comment]
@yuit can you review this?

**Pull request merged**